backend/app/router/endpoints/questions.py

```
@@ -2,48 +2,80 @@
-                "Gaussian wave packet",
-                "Polynomial x²",
-                "Lorentzian 1/(x²+1)"
-            ],
-            correct_answer=0
-        ),
-        Question(
-            question="Which theorem connects continuous symmetries to conservation laws in physics?",
-            answer="Noether's theorem"
-        )
-    ])
+    load_dotenv(dotenv_path=os.path.join(os.path.dirname(os.path.dirname(os.path.dirname(os.path.abspath(__file__)))), '.env'))
```

Pylint (3.3.7)

[convention] Line too long (127/100)

(C0301)